src/VisualStudio/VisualStudioDiagnosticsToolWindow/OptionPages/ForceLowMemoryMode.cs

```diff
@@ -0,0 +1,141 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Roslyn.VisualStudio.DiagnosticsWindow.OptionsPages
+{
+    internal class ForceLowMemoryMode
+    {
+        private int _size = 500;  // default to 500 MB
+        private MemoryHogger _hogger;
+
+        public static readonly ForceLowMemoryMode Instance = new ForceLowMemoryMode();
+
+        private ForceLowMemoryMode()
+        {
+        }
+
+        public int Size
+        {
+            get { return _size; }
+            set { _size = value; }
```

Comment:
No, you have to disable the low memory mode, change the value and re-enable to get it to hog a different amount of megabytes.